src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -2198,6 +2198,29 @@ TemporaryBattleList['Grand Duchess Diantha'] = new TemporaryBattle(
     }
 );
 
+TemporaryBattleList['Hex Maniac Aster'] = new TemporaryBattle(
+    'Hex Maniac Aster',
+    [
+        new GymPokemon('Gastly', 40250000, 25),
+        new GymPokemon('Haunter', 60770015, 30),
+        new GymPokemon('Haunter', 60770015, 30),
+        new GymPokemon('Medicham', 60770015, 30),
+    ],
+    'What’s this?! I see... Perhaps a Trainer as accomplished as you can get the most out of this.</br><img src="assets/images/megaStone/04.png"/></br>',
```

Comment:
Should ne 94, not 04

Reply:
fixed that and another typo - a missing full stop - in the next update.